lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -97,6 +97,7 @@ <h3 class="hd hd-3">${_("Reports")}</h3>
       <input type="button" name="calculate-grades-csv" class="async-report-btn" value="${_("Generate Grade Report")}" data-endpoint="${ section_data['calculate_grades_csv_url'] }"/>
       <input type="button" name="problem-grade-report" class="async-report-btn" value="${_("Generate Problem Grade Report")}" data-endpoint="${ section_data['problem_grade_report_url'] }"/>
       <input type="button" name="export-ora2-data" class="async-report-btn" value="${_("Generate ORA Data Report")}" data-endpoint="${ section_data['export_ora2_data_url'] }"/>
+      <input type="button" name="export-ora2-summary" class="async-report-btn" value="${_("Generate ORA Summary Report")}" data-endpoint="${ section_data['export_ora2_summary_url'] }"/>
```

Comment:
Worth mentioning in the PR description why we chose to put the button here instead of on the [Open Responses Dashboard](https://github.com/edx/edx-ora2/blob/f0520acfe21f88333587fb1514c1d8400e867148/openassessment/xblock/static/js/src/lms/oa_course_items_listing.js#L82) (so we can use the instructor task CSV generation logic)